packages/backend/src/peripherals/database/WithdrawableAssetRepository.ts

```diff
@@ -63,6 +66,19 @@ export class WithdrawableAssetRepository extends BaseRepository {
     return results[0]!.id
   }
 
+  async getAssetBalancesByStarkKey(
```

Comment:
I would actually like to rename the repository because WithdrawableAssetRespoitory doesn't contain withdrawable assets and then I would change the name to the initial one which was `getWithdrawableAssetsByStarkKey`

Reply:
What would you rename the repo to?